index.html

```diff
@@ -3,9 +3,7 @@
 
 <head>
   <meta charset="UTF-8" />
-  <link rel="icon" type="image/svg+xml" href="/tonomy-logo48.png" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Tonomy</title>
```

Comment:
shouldn't this also be set in settings.ts like the icon?

Reply:
it's already there document.title = settings.config.appName;